lib/srv/desktop/rdp/rdpclient/src/rdpdr.rs

```diff
@@ -2039,12 +2153,12 @@ impl DeviceAnnounceHeader {
 }
 
 #[derive(Debug)]
-struct ServerDeviceAnnounceResponse {
+struct ServerDeviceAnnounceResponseDeprecated {
```

Comment:
Why are all these structs renamed to `*Deprecated`?

Reply:
These are structs which exist from the rdp-rs era, but now a struct by the same name exists in IronRDP and we import it here. I think it will be easiest for me to deprecate them this way, while keeping the rdp-rs version of the code on hand for easy reference, and then rip out all the legacy code all at once at the end.

Reply:
Makes sense, thanks for clarifying